src/firebase-config-common.ts

```diff
@@ -16,6 +16,7 @@ export const getTypedFirestoreDataConverter = <T>(): CommonFirestoreDataConverte
 });
 
 export type SemesterDocumentData = {
```

Comment:
Can we define this as `Readonly<SemesterDocumentData>` where `SemesterDocumentData` refers to the mutable type defined in `src/firebase-admin-config`?

Reply:
I changed this in #724 btw